### PR TITLE
Horizontal scroll for sessions list

### PR DESCRIPTION
**Pull request by 3v0k4**
This just works for the mouse wheel. If users have trackpads or such things they should already be able to scroll horizontally.

It uses https://gist.github.com/pablen/c07afa6a69291d771699b0e8c91fe547

### CODE REVIEW

File: app/javascript/createObserver.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          const matchingElements = getMatchingElementsFromTree(
+            addedNode,
+            selector
+          );
+          if (matchingElements.length < 1) return;
+          matchingElements.forEach(node => onMount(node));
+        });
+      }
+      // Handle removed nodes
+      if (onUnmount) {
```
Comment (AkkarinForest) [inline app/javascript/createObserver.js]: Maybe we don't need to copy the entire file? eg looks like we are not using this if.

Comment (3v0k4) [reply app/javascript/createObserver.js]: We are not but it could be useful, especially if we use more of that

**Comment by 3v0k4** [comment]
Good catch. Haven't noticed that "wheel" is triggered on scroll too. It seems like there actually no good way to do that w/o a lot of hacking. I'll check back with others what to do here. Thanks!

**Comment by 3v0k4** [comment]
Checking `deltaX` should have solved the problem mentioned by @AkkarinForest

**Comment by gitjul** [comment]
👀 

thx for taking care of this <3

**Pull request merged**